sites/main-page-title/includes/main-page-title.html

```diff
@@ -0,0 +1,12 @@
+{% if include %}
+	{% if include.dir == 'ltr' or include.dir == 'rtl' %}
+		<h1 property="name"
+		{%- if include.rdfa -%}
+		typeof="WebPageElement"
+		{%- endif -%}
+		{%- if include.id -%}
+		id="wb-cont"
```

Comment:
I think that include is exclusively for the first level 1 heading in the content and I don't think that include should be used when there is additional level 1 heading through the page.

```suggestion
		id="wb-cont"
```